lib/twilio-ruby/rest/client.rb

```diff
@@ -71,6 +72,11 @@ def initialize(username=nil, password=nil, account_sid=nil, region=nil, http_cli
       # Makes a request to the Twilio API using the configured http client
       # Authentication information is automatically added if none is provided
       def request(host, port, method, uri, params={}, data={}, headers={}, auth=nil, timeout=nil)
+        whitelisted_numbers = ENV.fetch("WHITELISTED_NUMBERS", "").split
```

Comment:
The one thing I'm worried here is if you want to add your phone number to all the staging envs to all the applications, which are using Twilio directly, then you need to spend a decent amount of time :/

Reply:
This is a pretty hard thing to work around. The only thing I can suggest is:
1. We use a file instead
2. We write a quick script that is shard that makes updating the environment variable for each environment easier